src/app/models/project.ts

```diff
@@ -82,6 +83,7 @@ export class Project {
     this.details         = obj && obj.details         || undefined;
     this.engagementStatus = obj && obj.engagementStatus || undefined;
     this.logos            = obj && obj.logos || undefined;
+    this.shapefiles            = obj && obj.shapefiles || undefined;
```

Comment:
Just a little indentation weirdness.

Reply:
I'll see if I can get it fixed for this file!